howto/deploy-k8s-prod/README.md

```diff
@@ -0,0 +1,187 @@
+# Guidelines for production-ready deployments on Kubernetes
+
+This section outlines recommendations and practices for deploying Dapr to a Kubernetes cluster in a production-ready configuration.
+
+## Contents
+
+- [Cluster capacity requirements](#cluster-capacity-requirements)
+- [Sidecar resource requirements](#sidecar-resource-requirements)
+- [Deploying Dapr with Helm](#deploying-dapr-with-helm)
+- [Upgrading Dapr with Helm](#upgrading-dapr-with-helm)
+- [Recommended security configuration](#recommended-security-configuration)
+- [Tracing and metrics configuration](#tracing-and-metrics-configuration)
+
+## Cluster Capacity Requirements
+
+For a production ready Kubernetes cluster deployment, it is recommended you run a cluster of 3 nodes to support a highly-available setup of the control plane.
+The Dapr control plane pods are designed to be very lightweight and require the following resources in a production-ready setup:
+
+| Deployment  | CPU | Memory
+| ------------| ---- | ------
+| Operator | Limit: 1, Request: 100m | Limit: 200Mi, Request: 20Mi
+| Sidecar Injector  | Limit: 1, Request: 100m  | Limit: 200Mi, Request: 20Mi
+| Sentry  | Limit: 1, Request: 100m  | Limit: 200Mi, Request: 20Mi
+| Placement | Limit: 1, Request: 250m  | Limit: 500Mi, Request: 100Mi
+| Dashboard  | Limit: 200m, Request: 50m  | Limit: 200Mi, Request: 20Mi
+
+
+### Optional components
+
+The following control plane deployments are optional:
+
+* Placement - Needed for Dapr Actors
+* Sentry - Needed for mTLS for service to service invocation
+* Dashboard - Needed for operational view of the cluster
+
+## Sidecar Resource Requirements
+
+The Dapr sidecar requires the following resources in a production-ready setup:
+
+| CPU | Memory
+| --------- | --------- |
+| Limit: 4, Request: 100m | Limit: 4000Mi, Request: 250Mi
```

Comment:
Is this really 4 cores? I guess that is the upper limit.

Reply:
That's the upper limit.